doc/nrf/gs_modifying.rst

```diff
@@ -154,3 +154,75 @@ To specify CMake options, click :guilabel:`Tools` > :guilabel:`Options`, select
 If you work on the command line, pass the additional options to the ``west build`` command.
 The options must be added after a ``--`` at the end of the command.
 See :ref:`zephyr:west-building-cmake-args` for more information.
+
+.. _gs_modifying_build_types:
+
+Configuring build types
+***********************
+
+.. build_types_overview_start
+
+Build types enable you to use different sets of configuration options for each board.
+You can create several build type ``.conf`` files per board and select one of them when building the application.
+This means that you do not have to use one :file:`prj.conf` file for your project and modify it each time to fit your needs.
+
+.. build_types_overview_end
+
+.. note::
+    Creating build types and selecting them is optional.
+
+.. _gs_modifying_build_types_creating:
+
+Creating build type files
+=========================
+
+To create custom build type files for your application instead of using a single :file:`prj.conf` file, complete the following steps:
+
+1. During :ref:`application development <zephyr:application>`, follow the procedure for creating the application until after the step where you create the :file:`CMakeLists.txt` file.
```

Comment:
There is also https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_app_dev.html in the nrf repo, which I guess is a wrapper around Zephyr's Application development guide? So maybe link to that?

Reply:
I'll add a mention of build types there for sure, and can add an additional link.
Linking there instead of the current link can be confusing though, because we need to indicate a specific point where the build type files are to be created.